### DISCUSSION TITLE
Mocking redux and RTKQ

### DISCUSSION THREAD

**Discussion by S-Maitland** [storybookjs/storybook#26504] [Help] [answered]
### Summary

I have a story which is for a page. On this page there is a table that will be populated with data via an RTKQ. I don't need to simulate the calls however and I just want to be able to provide an array of fake objects to the story to populate the table. How do I go about this?

**Comment by johnemau** [comment, marked as answer]
I am in the same boat, I am considering using MSW (Mock Service Worker) as recommended by the docs here: https://redux.js.org/usage/writing-tests#ui-and-network-testing-tools

**Reply by ragraff** [reply]
Have used MSW for 2 projects now in the last year, it is pretty easy to use and effective, would recommend again
